parm/FV3LAM_wflow.xml

```diff
@@ -2023,7 +2033,7 @@ MODULES_RUN_TASK_FP script.
     &RSRV_ENKF;
     &WALL_LIMIT_RECENTER;
 
-    <command>&LOAD_MODULES_RUN_TASK_FP; "&RUN_RECENTER_TN;" "&JOBSdir;/JRRFS_CALC_ENSMEAN"</command>
+    <command>&LOAD_MODULES_RUN_TASK_FP; "&RUN_PREPSTART_TN;" "&JOBSdir;/JRRFS_CALC_ENSMEAN"</command>
     &RESOURCES_RUN_RECENTER;
```

Comment:
From Line 1238 to Line 2026:  We need to keep what is in the latest dev-sci branch.